docs/testing-operators.md

```diff
@@ -297,12 +259,12 @@ spec:
   channel: <channel-name>
   name: my-operator
   source: johndoe-operators
-  sourceNamespace: olm
+  sourceNamespace: marketplace
 ```
 
 If your Operator supports watching all namespaces, change the namespace of the Subscription from `default` to `operators`.
```

Comment:
what do you think about adding a not highlighting that `<channel-name>` has to be replaced by the `channels.name` in the package YAML file for the operator?

Reply:
also as for the other sections, adding something like ...

Deploy the `Subscription` resource:
```
kubectl apply -f operator-subscription.yaml
```